README.md

```diff
@@ -8,13 +8,38 @@
 **Table of Contents**
 
 - [Installation](#installation)
+- [Configuration](#configuration)
 - [License](#license)
 
 ## Installation
+This project is meant to run as a plugin within a Nebari deployment. To learn how to get started with Nebari, check out the docs [here](https://www.nebari.dev/docs/welcome).
 
-```console
-pip install nebari-plugin-self-registration
-```
+In order to install this plugin as part of a Nebari deployment:
+- Create a conda environment for your Nebari deployment
+- Install the self registration plugin with `pip install nebari-plugin-self-registration`
+- Continue the initialization and deployment of Nebari per your provider [instructions](https://www.nebari.dev/docs/explanations/provider-configuration).
+
+> **NOTE:** When running `nebari render` and `nebari deploy`, Nebari will detect and install any extensions which are installed in your Python environment.  When managing multiple Nebari deployments, be sure to manage your conda environments to ensure the correct extensions and versions are installed in your target deployment.
+
+
+### Configuration
+The configuration of your self registration app can be customized in several ways by creating a `config.yaml` file within the `self-registration` folder. An example of a configuration yaml is provided [here](https://github.com/MetroStar/nebari-self-registration/blob/main/self-registration/config.sample.yaml).
```

Comment:
Where should be the `self-registration` folder? Same level as `nebari-config.yaml`?

When I installed the plugin, I added a section like this to the `nebari-config.yaml` file:
```yaml
self_registration:
  namespace: self-registration
  coupons:
    - abcdefg
  approved_domains:
    - metrostar.com
  account_expiration_days: 30
  registration_group: test-group
```

Would it be worth to add this as another option in the docs for users that prefer to keep all the configuration within a single file?